.bazelversion

```diff
@@ -1 +1 @@
-2.0.0
+2.1.0
```

Comment:
revert?

Reply:
done; I'm going to have to be careful about this as recently I've found I can't build locally without that change.